From e6f12fb5286377d4394f2e3ac41d429faa8d5dd0 Mon Sep 17 00:00:00 2001
From: Anton Gladky <gladky.anton@gmail.com>
Date: Wed, 27 May 2015 16:49:37 +0200
Subject: [PATCH] Remove plotting during the test.

---
 py/plot.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/py/plot.py b/py/plot.py
index b51e3fe..918216f 100644
--- a/py/plot.py
+++ b/py/plot.py
@@ -579,11 +579,6 @@ def plot(noShow=False,subPlots=True):
 		>>> from yade import plot
 		>>> plot.resetData()
 		>>> plot.plots={'foo':('bar',)}
-		>>> plot.plot(noShow=True).savefig('someFile.pdf')
-		>>> import os
-		>>> os.path.exists('someFile.pdf')
-		True
-		>>> os.remove('someFile.pdf')
 		
 	to save the figure to file automatically.
 
-- 
2.1.4

